tests/core/test_views_swagger.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+import mock
+import yaml
```

Comment:
Yes, we should. This should have raised on tests, but I guess that's another package that imports pyYAML.